tools/svg2icns.sh

```diff
@@ -0,0 +1,90 @@
+# Script to convert SVG files to macOS .icns format.
+# Adapted from https://github.com/magnusviri/svg2icns.
```

Comment:
Could you copy the copyright + license from the initial script here for reference?